c/tests/test_tables.c

```diff
@@ -10420,6 +10421,20 @@ test_table_collection_delete_older(void)
     tsk_treeseq_free(&ts);
 }
 
+static void
+test_table_collection_edge_diffs_no_index(void)
+{
+    int ret;
+    tsk_table_collection_t tables;
+    tsk_diff_iter_t iter;
+    ret = tsk_table_collection_init(&tables, 0);
+    CU_ASSERT_EQUAL(ret, 0);
+    ret = tsk_edge_table_add_row(&tables.edges, 0., 1., 0, 1, NULL, 0);
+    CU_ASSERT_EQUAL(ret, 0);
+    ret = tsk_diff_iter_init_tables(&iter, &tables, 0);
+    CU_ASSERT_EQUAL(ret, TSK_ERR_TABLES_NOT_INDEXED);
```

Comment:
Need to free the tables here or the valgrind tests will fail.

Reply:
Best free the diff_iter too I guess

Reply:
Done.